eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -41,12 +41,12 @@ class InstallPlatformCommand extends Command
     const EXIT_MISSING_PERMISSIONS = 7;
 
     public function __construct(
-        Connection $db,
+        Connection $connection,
         array $installers,
         CacheItemPoolInterface $cachePool,
         $environment
     ) {
-        $this->db = $db;
+        $this->connection = $connection;
         $this->installers = $installers;
         $this->cachePool = $cachePool;
         $this->environment = $environment;
```

Comment:
If you touched class property maybe it is worth to add `string` as a type hint for `$this->environment`

Reply:
Good catch, fixed.